app/grandchallenge/algorithms/templates/algorithms/job_detail.html

```diff
@@ -52,10 +52,9 @@
                    aria-selected="false"><i class="fas fa-terminal fa-fw"></i>&nbsp;Logs
                 </a>
             {% endif %}
-
             {% if object.status == object.SUCCESS and perms.reader_studies.add_readerstudy %}
                 <a class="nav-link"
-                   href="{% url "algorithms:display-set-from-job-create" slug=object.algorithm_image.algorithm.slug interface_pk=object.pk %}">
+                   href="{% url "algorithms:display-set-from-job-create" slug=object.algorithm_image.algorithm.slug pk=object.pk %}">
                     <i class="fas fa-edit fa-fw"></i>&nbsp;Edit in Reader Study
```

Comment:
I had accidentally changed this from `pk` to `interface_pk` in one of my earlier PRs to the feature branch. It should stay at `pk` (refers to job pk).